README.md

```diff
@@ -1 +1,34 @@
 # Line Ending Selector [![Build Status](https://travis-ci.org/atom/line-ending-selector.svg?branch=master)](https://travis-ci.org/atom/line-ending-selector)
+
+![status bar tile](https://cloud.githubusercontent.com/assets/1305617/9274149/6b317568-4293-11e5-83ba-614a6c0d9890.png)
+
+
+This is an [Atom](https://atom.io) package that displays the current line ending type of a file: `CRLF` (Windows), `LF` (Unix), or `Mixed` (both). It also lets the user change the line ending of a file.
```

Comment:
I would probably change `lets the user` to `lets you`, since you're addressing the reader directly elsewhere.